src/definitions.ts

```diff
@@ -269,9 +269,12 @@ export class NEARBindingsBuilder extends ExportsWalker {
   }
 
   private generateHandlerMethods(valuePrefix: string, fields: any[]) : void {
-    function fieldsWithTypes(types: string[]) {
-      return fields.filter(field => types.indexOf(field.type.toString()) != -1);
-    }
+    let fieldsWithTypes = (types: string[]): any[] => {
```

Comment:
@bowenwang1996 nit: I think. you can have more compact code:
```
let fieldsWithTypes = (types: string[]): any[] =>
    fields.filter(field => types.indexOf(this.typeName(field.type)) != -1);
```

Reply:
Fixed on master